### PR TITLE
Inclusion of new category Travel

### PR DESCRIPTION
**Pull request by psqaure11**

### CODE REVIEW

File: customQuotes/category.json [hunk 1]
```diff
@@ -788,5 +788,27 @@
...
+    },
+    {
+      "quote": "The journey of a thousand miles begins with one step.",
+      "author": "Lao Tzu"
+    },
+    {
+      "quote": "A journey is best measured in friends, not in miles. ",
+      "author": "Tim Cahill"
+    }
+  ]}
```
Comment (shravan20) [inline customQuotes/category.json:814]: Add a bnew le
```suggestion
   ]
  }
  
```

**Review by shravan20 — CHANGES_REQUESTED** [review]
PR LGTM, one change suggested

File: customQuotes/category.json (hunk 1 above)
Comment (psqaure11) [reply customQuotes/category.json]: I have made the necessary changes

**Comment by psqaure11** [comment]
Shravan if you have some time Today kindly look into the changes I have made and merge them

**Review by shravan20 — APPROVED** [review]
PR LGTM

**Comment by shravan20** [comment]
@psqaure11: Thank you for your contribution

**Pull request merged**